src/app/plugins/stock.plugin.ts

```diff
@@ -101,6 +172,28 @@ export class StockPlugin extends Plugin {
     return embed;
   }
 
+  private _makeCryptoEmbed(quote: any) {
```

Comment:
ideally, you would want to expand on `_makeEmbed()` instead of making a whole new method. maybe passing in an enum for what type of embed could be a better pattern at tackling this